test/alternator/test_tablets.py

```diff
@@ -84,6 +91,37 @@ def test_initial_tablets_number(dynamodb):
     with new_test_table(dynamodb, **schema) as table:
         assert not uses_tablets(dynamodb, table)
 
+# Usage of tablets is determined by the configuration flag
+# "tablets_mode_for_new_keyspaces", as well as by the per-table
+# "system:initial_tablets" tag. The tag overrides the configuration,
+# except when the configuration flag's value is "enforced" -
+# then if the tag asks for vnodes, an exception is thrown.
+# The test checks only 2 of the values of the config flag
+# "tablets_mode_for_new_keyspaces": "enabled" and "enforced". The value
+# "disabled" is likely never to be used in the wild and additionally
```

Comment:
I think instead of explaining why you don't want to test "disabled" mode, you should just test it - it's easy.